src/backends/torch/torchdataset.cc

```diff
@@ -119,11 +119,23 @@ namespace dd
   }
 
   void TorchDataset::image_to_stringstream(const cv::Mat &img,
-                                           std::ostringstream &dstream)
+                                           std::ostringstream &dstream,
+                                           const bool &compression)
```

Comment:
I'd call it "lossless", in both cases the image is compressed, but with jpeg the image can be altered, while with png the image remains exactly the same

Reply:
In practice PNG is an order of magnitude bigger than JPG, very impacting on datasets with millions of images. Additionnally it doesn't matter the image is compressed for training, since the neural net much often would need to accomodate that type of distorsion.
Ground truth, aka masks however must not be compressed.

Reply:
I do agree with what you say, that's why I asked for keeping images compressed as jpg.
However there is a vocabulary misunderstanding here:
masks must not be *altered*, however we can *compress* them.
PNG *is* a compressed format, as opposed to bitmap, but it's a *lossless* one, which means it reduces the size of the image without altering it at all. In another hand, jpg produces much smaller images but alters the image, so we can not use it for masks.

In both cases the image *is* compressed, so the parameter name `compression` is misleading, it would be preferable to rename it `lossless`.

Reply:
OK!